Dockerfile

```
@@ -5,6 +5,7 @@ ARG NIMFLAGS
 ARG MAKE_TARGET=wakunode2
 ARG NIM_COMMIT
 ARG LOG_LEVEL=TRACE
+ARG HEAPTRACK_BUILD=0
```

```suggestion
ARG HEAPTRACK_BUILD=false
```

bit weird to use `0` IMO.

---

We use `0` or `1` for compatibility with all possible Makefile settings. This should be kept `0` in that PR and open a separate PR to perform the transition from `0`/ `1` to `false`/`true`, which might have implication in CI files.